src/components/tutorials/advanced-tutorial.js

```diff
@@ -28,8 +28,9 @@ class AdvancedTutorial extends React.Component {
             </a>
             </h2>
             <p>Previously, we created a submission and added a measurement set in two requests. It's common to want to do these at the same time - the first time we want to submit performance data for any individual, for instance. It's convenient to be able to do both, so let's try that with ACI performance data.</p>
-            <p>Here's a <code>POST</code> request to create a submission. We're including the  measurement set and submission together by nesting the measurement set inside the submission info, much like we nested measurements within measurement sets.</p>
-            <p>The request payload on the right can be harder to read, but it contains the same info as below formatted exactly as it's sent through the API. Check it out then click the button!</p>
+            <p>Here's a <code>POST</code> request to create a submission. Again, you can pass an <code>Accept</code> header to specify the API version and type of response back. We use a custom mime type, <code>application/vnd.qpp.cms.gov.v1+json</code>, and support JSON and XML. Right now, only <code>v1</code> is supported. You can also continue to use a standard <code>application/json</code> or <code>application/xml</code> header.</p>
```

Comment:
> and type of response back

maybe better as `and the desired response format`

> We use a custom mime type, <code>application/vnd.qpp.cms.gov.v1+json</code>

might be helpful to link to literature about custom mime types here, and note that the +json can be swapped for +xml, I'm not sure that's clear from this sentence.

> You can also continue to use a standard <code>application/json</code> or <code>application/xml</code> header.

We should specify that these will always point to the latest version (which is currently v1)

Reply:
Looks good to me! I think as this is the advanced one, then you can be more specific/technical with the language.

Reply:
👍 not sure about linking to external sites to content we don't control. i think the added explanation is good enough for ppl to understand how to use. 

maybe a good topic for a lightning blog post?